.github/workflows/conda_meta.yml

```diff
@@ -29,7 +32,7 @@ jobs:
         shell: bash -l -e {0}
         run: |
           conda info
-          conda env update -f skare3/pkg_defs/ska3-core-latest/environment.yml
+          conda env update -f skare3/pkg_defs/ska3-core-latest/base_environment.yml
```

Comment:
The file in there is a 3.10 file -  not sure if it matters or if there's an uncommited change you have hanging around which updates it.

Reply:
That file is the one used to make ska3-core from scratch. It is updated in #1220 

By the way, the changes in this PR are also in #1220. I created a branch like the 2024.0-branch, and rebased interactively to tidy it up.

Reply:
You've lost me a bit. I thought this PR could be reviewed independently because it is against master and master has a 3.10 version of the base_environment file and there are no changes to it in this PR.  So ...

Reply:
I thought even if this is basically a PR to capture workflow updates that it would be useful to know that all the files at speedy-build would actually work together.